airflow/cli/commands/remote_commands/config_command.py

```diff
@@ -227,6 +227,21 @@ def message(self) -> str:
         suggestion="Remove TaskContextLogger: Replaced by the Log table for better handling of task log "
         "messages outside the execution context.",
     ),
+    ConfigChange(
+        config=ConfigParameter("logging", "log_filename_template"),
+        suggestion=(
+            "The default value in Airflow 2 (e.g., '{{ ti.dag_id }}/{{ ti.task_id }}/{{ ts }}/{{ try_number }}.log') "
+            "breaks task logs in the new UI. Please update to the new recommended format as per "
+            "https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#log-filename-template."
+        ),
+        was_deprecated=False,
+    ),
+    ConfigChange(
+        config=ConfigParameter("logging", "log_processor_filename_template"),
```

Comment:
Looks down a few lines, this config is already removed.

Reply:
removed it